src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionsLoaderTest.java

```diff
@@ -305,4 +307,50 @@ public void testSettingModuleId() throws Exception {
         final String id = Whitebox.getInternalState(suppressElement, "moduleId");
         assertEquals("Id has to be defined", "someId", id);
     }
+
+    @Test
+    public void testXpathSuppressions() throws Exception {
+        final String fn = getPath("suppressions_xpath_correct.xml");
```

Comment:
All new inputs should be named after their tests.
The file name should start with `InputSuppressionsLoader`.
Please do this for all new inputs.

Reply:
done